draft-ietf-ppm-dap.md

```diff
@@ -110,6 +110,40 @@ possible by distributing the computation among the servers in such a way that,
 as long as at least one of them executes the protocol honestly, no input is ever
 seen in the clear by any server.
 
+## Change Log
+
+(\*) Indicates a change that breaks wire compatibility with the previous draft.
+
+02:
+
+- Define a new task configuration parameter, called the "query type", that
+  allows tasks to partition reports into batches in different ways. In the
+  current draft, the Colletor specifies a "query", which the Aggregators use to
+  guide selection of the batch. Two query types are defined: the "time-interval"
+  type captures the semantics of draft 01; and the "fixed-size" type allows the
+  Leader to partition the reports arbitrarily, subject to the constraint that
+  each batch is roughly the same size. (\*)
+
+- Define a new task configuration parameter, called the task "expiration", that
+  defines the lifetime of a given task.
+
+- Specify requirements for HTTP request authentication rather than a concrete
+  scheme. (Draft 01 required the use of bearer tokens; these are now optional.)
```

Comment:
```suggestion
  scheme. (Draft 01 required the use of the `DAP-Auth-Token` header; this is now optional.)
```
To be pedantic, we didn't require bearer tokens in draft-01. In HTTP, a bearer token is when you have an `Authorization` header whose value is like `Bearer <token>`.

https://datatracker.ietf.org/doc/html/rfc6750#section-2.1
https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication